ios/ReactTestApp/ReactTestApp.common.xcconfig

```diff
@@ -18,3 +18,17 @@ OTHER_LDFLAGS = $(inherited) -fstack-protector-strong
 SDKROOT = iphoneos
 SWIFT_TREAT_WARNINGS_AS_ERRORS = YES
 WARNING_CFLAGS = -Wall
+
+RUN_CLANG_STATIC_ANALYZER = YES
+CLANG_STATIC_ANALYZER_MODE = deep
+
+// Required security settings for production code (do not override at target/project level, with the possible
```

Comment:
Does this mean that it's only required for code that gets shipped to customers? `react-native-test-app` does not get shipped to customers, and these flags only get applied to the test app itself. Do we still need them here?

Reply:
I intended to add the flags here so they get injected into all targets (I.E: The individual native modules that a project's test app is building). It sounds like I actually want to inject them into `user_target_xcconfig`. I'm not sure if `xcodebuild analyze` is doing something different from these flags. @harrieshin / @anandrajeswaran any opinions?